lib/Controller/Task.php

```diff
@@ -490,13 +488,13 @@ public function run(Request $request, Response $response, $id)
             $task->lastRunStatus = \Xibo\Entity\Task::$STATUS_ERROR;
         }
 
-        $task->lastRunDt = $this->getDate()->getLocalDate(null, 'U');
+        $task->lastRunDt = Carbon::createFromTimestamp(time())->format('U');
```

Comment:
The more I look through the more I think replacing this with:


```suggestion
        $task->lastRunDt = Carbon::now()->format('U');
```